components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/constant/WebSubHubAdapterConstants.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.event.websubhub.publisher.constant;
+
+/**
+ * Keep constants required by the WebSubHub Event Adapter.
+ */
+public class WebSubHubAdapterConstants {
+
+    public static final String CONFIG_FILE_NAME = "identity-outbound-adapter.properties";
```

Comment:
Grouping the constants, identifying the enum types will be helpfull.

Reply:
Fixed with https://github.com/wso2-extensions/identity-event-publishers/pull/1/commits/eb4cae044fe65111cb72917fd78af0619d056a02